.github/workflows/docs.yaml

```diff
@@ -3,10 +3,10 @@ name: Sphinx Docs
 on:
   push:
     branches:
-      - documentation
+      - main
   pull_request:
     branches:
-      - documentation
+      - main
```

Comment:
this is imho wrong.
please double check @schuenke 

will this run on all pull request made against main and overwrite the public documenation?

it should not run the publish to github pages on PR!

Reply:
Yes, it runs on pushes and pull requests against main. I though this is useful to see the final result (at least for this PR). Removed the pull_request dependency.